src/react/components/common/tagInput/tagInputToolbar.tsx

```diff
@@ -57,6 +58,12 @@ export default class TagInputToolbar extends React.Component<ITagInputToolbarPro
                 category: Categories.General,
                 handler: this.handleAdd,
             },
+            {
+                displayName: strings.tags.toolbar.onlyShowCurrentPageTags,
+                icon: "View",
+                category: Categories.General,
+                handler: this.handleOnlyCurrentPageTags,
+            },
```

Comment:
Suggestion - when we're hiding unused tags we could use this icon: 
![image](https://user-images.githubusercontent.com/64093224/95681407-8509a800-0b94-11eb-96f9-b34e805d5959.png) for more clear visual clue. So ```View``` icon when when all tags are visible, and ```Hide3``` icon when we're hiding unused tags.

Reply:
icon changed

Reply:
I think it would be better to use something like the Filter and ClearFilter icons:
![image](https://user-images.githubusercontent.com/60453211/95732020-16305b80-0c35-11eb-8066-28acf3f6e3e7.png)
Since we're filtering for tags with values. We already use the hide and show icons for password protection.